addons/project/models/project.py

```diff
@@ -1318,6 +1320,8 @@ def _compute_partner_phone(self):
         for task in self:
             if task.partner_id and task.partner_phone != task.partner_id.phone:
                 task.partner_phone = task.partner_id.phone
+            elif not task.partner_id:
+                task.partner_phone = ''
```

Comment:
Why don't we do the same for the partner_email field? 🙂

Reply:
Moreover, I'm not sure but when we have no `partner_id` then the `task.partner_id.phone` is equal to an empty string, isn't it? 🤔 
If it is the then we could just do this:
 ```py
    @api.depends('partner_id.phone')
    def _compute_partner_phone(self):
        for task in self:
            if task.partner_phone != task.partner_id.phone:
                task.partner_phone = task.partner_id.phone
```
I am really not sure for this. ^^
What do you think? 🙂

Reply:
It worked, I changed it for both the email and the phone :+1: